src/core.js

```diff
@@ -1277,6 +1277,7 @@ define([
         }
         this.data.core.form = form = parser.parseXForm(
             formXML, options, this, _this.data.core.parseWarnings);
+        this.onXFormLoaded(form);
```

Comment:
Had to add this new extension point so javaRosa could load all data references in `<output>` tags before the tree is populated.